packages/react-server-dom-webpack/src/ReactFlightClientWebpackBundlerConfig.js

```diff
@@ -56,7 +60,9 @@ const asyncModuleCache: Map<string, Thenable<any>> = new Map();
 
 // Start preloading the modules since we might need them soon.
 // This function doesn't suspend.
-export function preloadModule<T>(moduleData: ModuleReference<T>): void {
+export function preloadModule<T>(
+  moduleData: ModuleReference<T>,
+): null | Thenable<any> {
```

Comment:
This changes the protocol of preloading modules. It has to suspend here and not in require module. This is not updated in ReactFlightDOMRelayClientIntegration because they live in www. So someone else has to update it.

https://github.com/facebook/react/blob/main/packages/react-server-dom-relay/src/__mocks__/ReactFlightDOMRelayClientIntegration.js#L16-L19

I don't think it's currently used there so it's not blocking.

The protocol is that `requireModule` can no longer suspend by throwing a Promise. Instead, `preloadModule` should return a Promise.

@frandiox this will also affect the Vite parts.

Reply:
cc @kassens